app/AcquireIntersonArrayRF.cxx

```diff
@@ -121,18 +121,18 @@ int main( int argc, char * argv[] )
   ContainerType container;
 
   const int height = hwControls.GetLinesPerArray();
+  std::cout << "Lines per array: " << height << std::endl;
   const int width = ContainerType::MAX_RFSAMPLES;
```

Comment:
Here width is declared as MAX_RFSAMPLES and the height is number of lines.

In IntersonArrayCxxImagingContainer.h, Line 344, this commit defines width as NBOFLINES (i.e., LinesPerArray) and the height as the MAX_RFSAMPLES.

Which is correct?    

This was a point of confusion for me as well.  I believe the width (i.e., the 0-dimension coordinate) is MAX_RFSAMPLES and the height is the NBOFLINES (as stated here, but not as stated on line 344 in the other file).